server/command.go

```diff
@@ -385,6 +405,7 @@ func executeUninstallCloud(p *Plugin, c *plugin.Context, header *model.CommandAr
 		wSEventInstanceStatus,
 		map[string]interface{}{
 			"instance_installed": false,
+			"instance_type":      "",
```

Comment:
0/5 this seems unnecessary - promotes reliance on non-nil values on the webapp side?

Reply:
@levb The checks that the webapp work like this now:
`isInstanceInstalled(state) && getInstalledInstanceType(state) === 'server'`

I don't see much benefit of including or excluding this value. Though, I think it's good to be consistent with these fields that are coupled together.

Reply:
@levb javascript treats the empty string as falsy so it doesn't make a difference if you were to do something like

`if (!installedInstanceType) {`

That would cover the case of not present, null and ""

Reply:
Then why set it in the map at all, on the server?

Reply:
It's part of the data model, and this ensures we won't ever have an issue with inconsistent stale data like
```
{"instance_installed": false, "instance_type": "server"}
```

I'm just making sure to stay consistent and always make sure the two keys agree with each other, as the consumer of the data could process it in any given way. If you think it truly shouldn't be there, I can remove it.